backend/src/Wasm/Libs/WASM.fs

```diff
@@ -68,6 +246,115 @@ let fns : List<BuiltInFn> =
         | _ -> incorrectArgs ())
       sqlSpec = NotQueryable
       previewable = Impure
+      deprecated = NotDeprecated }
+
+
+    { name = fn "WASM" "requestFromWasm" 0
```

Comment:
This function should probably be renamed?

Reply:
fyi we allow multiple modules now. I think you can do `name = fn' ["Wasm";"HttpClient";"Even";"More";"If";"Desired"] "request" 0`